ibmcloud/image/push.sh

```diff
@@ -82,10 +82,16 @@ while image_id=$(ibmcloud is images --output=json --visibility=private | jq -r "
 done
 
 echo -e "\nUploading $image_path to cloud object stroage at "$cos_bucket" with key $object_key\n"
-ibmcloud cos object-put  --bucket "$cos_bucket" --key "$object_key" --body "$image_path"
+./multipart_upload.sh --bucket "$cos_bucket" --file "$image_path"
+
 ibmcloud cos object-head --bucket "$cos_bucket" --key "$object_key"
 
-image_ref="cos://$region/$cos_bucket/$object_key"
+if [[ ! "$region" =~ [A-Za-z]+-[A-Za-z]+ ]]; then 
+    location="${region}-geo"
+else 
+    location=$region
+fi
```

Comment:
You can simply write the same logic like this.
```
case "$region" in
    *-*) location=$region;;
    *)   location="${region}-geo";;
esac
```

BTW, I only see `eu-geo` defined as this pattern in https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints&locale=en.

Does this code also work with regions like `us` and `ap`?

Reply:
Yes the -geo logic is the same for `us`, and `ap` too. 
![image](https://user-images.githubusercontent.com/105913324/185571174-8d5801a3-cc8b-4e32-a557-b123b2e38acc.png)

Reply:
I see. Thank you for the info.

I guess what you wanted to code is something like this.

```
if [[ ! "$region" =~ ^[A-Za-z]+-[A-Za-z]+$ ]]; then
```

Otherwise, just this one.

```
if [[ ! "$region" =~ - ]]; then
```